internal/ansible/controller/reconcile_test.go

```diff
@@ -35,6 +35,10 @@ import (
 	"github.com/operator-framework/ansible-operator-plugins/internal/ansible/runner/fake"
 )
 
+// The behaviour of fake client has changed with
+// status subreasources (ref: https://github.com/kubernetes-sigs/controller-runtime/pull/2259).
+// (Tech Debt) This should be re written to use envtest to avoid any more breaking
+// complications in future which include removal of flake client.
```

Comment:
Can we capture this in an issue?

Reply:
Captured here: https://github.com/operator-framework/ansible-operator-plugins/issues/38